catalog/transformer/catalog-transformer-csv-queryresponse/src/test/java/ddf/catalog/transformer/csv/CsvMetacardTransformerTest.java

```diff
@@ -45,71 +44,69 @@
-                    buildAttributeDescriptor("doubleAtt", BasicTypes.DOUBLE_TYPE));
-
-    private static final List<Attribute> ATTRIBUTES =
-            ImmutableList.of(
-                    new AttributeImpl("stringAtt", "stringVal"),
-                    new AttributeImpl("intAtt", 101),
-                    new AttributeImpl("doubleAtt", 3.14159));
-
-    @Before
-    public void setUp() {
-        this.transformer = new CsvMetacardTransformer();
-        this.arguments = new HashMap<>();
-        String[] columnOrderArray = {"stringAtt", "intAtt", "doubleAtt"};
-        arguments.put("columnOrder", buildList(columnOrderArray));
-        normalMC = buildMetacard();
-    }
-
-    private static AttributeDescriptor buildAttributeDescriptor(String name, AttributeType type) {
-        return new AttributeDescriptorImpl(name, true, true, true, true, type);
-    }
-
-    private ArrayList<String> buildList(String[] entries) {
-        ArrayList<String> arrayList = new ArrayList<>();
-        arrayList.addAll(Arrays.asList(entries));
-        return arrayList;
-    }
-
-    @Test(expected = CatalogTransformerException.class)
-    public void testTransformerWithNullMetacard() throws CatalogTransformerException {
-        transformer.transform(nullMC, arguments);
-    }
-
-    @Test
-    public void testCsvMetacardTransformer() throws CatalogTransformerException, IOException {
-        BinaryContent binaryContent = transformer.transform(normalMC, arguments);
-        assertThat(binaryContent.getMimeType().toString(), is("text/csv"));
-
-        List<String> attributes = Arrays.asList(new String(binaryContent.getByteArray()).split("\r\n"));
-        List<String> attNames = Arrays.asList(attributes.get(0).split(","));
-        List<String> attValues = Arrays.asList(attributes.get(1).split(","));
-
-        for (int i = 0; i < attNames.size(); i++) {
-            String attributeValue = attValues.get(i);
-            assertThat(
-                    attributeValue.equals(ATTRIBUTES.get(i).getValue().toString()),
-                    is(true));
-        }
+  Map<String, Serializable> arguments;
+
+  private CsvMetacardTransformer transformer;
+
+  Metacard normalMC, nullMC;
+
+  private static final Set<AttributeDescriptor> ATTRIBUTE_DESCRIPTORS =
+      ImmutableSet.of(
+          buildAttributeDescriptor("stringAtt", BasicTypes.STRING_TYPE),
+          buildAttributeDescriptor("intAtt", BasicTypes.INTEGER_TYPE),
+          buildAttributeDescriptor("doubleAtt", BasicTypes.DOUBLE_TYPE));
+
+  private static final List<Attribute> ATTRIBUTES =
+      ImmutableList.of(
+          new AttributeImpl("stringAtt", "stringVal"),
+          new AttributeImpl("intAtt", 101),
+          new AttributeImpl("doubleAtt", 3.14159));
+
+  @Before
+  public void setUp() {
+    this.transformer = new CsvMetacardTransformer();
+    this.arguments = new HashMap<>();
+    String[] columnOrderArray = {"stringAtt", "intAtt", "doubleAtt"};
```

Comment:
❓Is there a reason that we don't just make this an `ArrayList` to begin with? And get rid of the `buildList` function?